ecosystem/sep-0008.md

```diff
@@ -3,54 +3,65 @@
 ```
 SEP: 0008
 Title: Regulated Assets
-Author: Interstellar.com
+Author: Interstellar.com, Howard Chen <@howardtw>
 Status: Active
 Created: 2018-08-22
-Updated: 2020-12-09
-Version 1.0.0
+Updated: 2020-12-10
+Version 2.0.0
 ```
 
 ## Simple Summary
 
-Regulated Assets provide ecosystem support for assets that require an issuer’s approval (or a delegated third party’s approval, such as a licensed securities exchange) on a per-transaction basis. It standardizes the identification of such assets as well as defines the protocol for performing compliance checks and requesting issuer approval.
+Regulated Assets are assets that require an issuer’s approval (or a delegated third party’s approval, such as a licensed securities exchange) on a per-transaction basis. It standardizes the identification of such assets as well as defines the protocol for performing compliance checks and requesting issuer approval.
 
-**Target Audience**: Asset issuers, issuance platforms, wallet developers, and exchanges
+## Target Audience
+
+Asset issuers, issuance platforms, wallet developers, and exchanges.
 
 ## Motivation
 
-Stellar aims to be an ideal platform for issuing securities. As such, it must provide tools to comply with various regulatory requirements. Regulation often requires asset issuers to monitor and approve every transaction involving the issued asset and to enforce an assortment of constraints. This SEP provides support for these requirements.
+Stellar is an ideal platform for issuing securities. Regulation sometimes requires asset issuers to monitor and approve each transaction involving issued assets and to enforce constraints. This is possible on the Stellar network today, however there exists no standard interface between wallets and issuers to negotiate approval.
 
 ## Overview
 
-Implementing a Regulated Asset consists of these parts:
-- **Stellar.toml**: Issuers will advertise the existence of an Approval Service and the approval criteria via their stellar.toml file.
-- **Approval Server**: HTTP protocol for transaction signing.
-- **Account Setup**: Wallets will have to work with accounts that are controlled or at least partially controlled (via multisig) by asset issuers, rather than the wallet end user.<br/>
-**Note**: this step may not be required once the proposed [protocol change](https://github.com/stellar/stellar-protocol/issues/146) allowing for protocol-level per-transaction approval is implemented.
+Implementing a regulated asset consists of these parts:
+- [Identification]: Regulated asset issuers set the [authorization flags] in the asset issuing account. Regulated asset issuers advertise the existence of an approval server and the approval criteria via their [SEP-1 stellar.toml] file.
+- [Transaction Composition]: Transactions consist of operations that authorize the holding account, transact the regulated asset, and deauthorize the holding account.
```

Comment:
❗ `holding account` appears to be a new term in this SEP that isn't well defined. Could we use the term client's account here to indicate whose account it is, or transacting account to describe which account it is?

Reply:
I've removed this term in https://github.com/stellar/stellar-protocol/pull/817/